domain/ide/workspace/folder.go

```diff
@@ -262,7 +262,7 @@ func NewFolder(
 
 func (f *Folder) sendEmptyDiagnosticForFile(path string) {
 	config.CurrentConfig().Logger().Debug().Str("filePath", path).Msg("sending empty diagnostic for file")
-	f.sendDiagnosticsForFile(path, []snyk.Issue{})
+	f.sendDiagnosticsForFile(product.ProductUnknown, path, []snyk.Issue{})
```

Comment:
this is wrong. The caller has the issue Type and it should be passed down to sendEmptyDiagnosticForFile